app/assets/javascripts/miq_flash.js

```diff
@@ -35,6 +39,17 @@ function add_flash(msg, level, options) {
   var alertDiv = $('<div class="' + cls.alert + '"><button class="close" data-dismiss="alert"><span class="pficon pficon-close"></span></button></div>');
   alertDiv.append(iconSpan, textStrong);
 
+  // if options.long alert is given than add a `See More` button to the alert div to allow user the get more details of the error/alert.
+  if (options.long_alert) {
+    var detailsLinkTxt = _("View More");
```

Comment:
do you add the "View More" even if msg is short ?

e.g.
```
short error msg. View More
```

Reply:
@yaacov yep. We discussed it with @himdel before.. I agree it's not ideal.

Reply:
Than why not check the length  ?

Reply:
nice, @himdel agreed to show a do-nothing "View More"/"View Less" links, that only change from "More" to "Less" when pressed :-) , I will remember that one next time he nitpick me on something.

Reply:
@yaacov I think I will check the length eventually.. I just want @himdel to agree to it too cause I know he has issues with this approach and he's the one that matters 😝

Reply:
This is now addressed by [checkEllipsis](https://github.com/ManageIQ/manageiq-ui-classic/pull/2862/files#diff-c31814e321dfd0b05558e69e59e39e68R75), right?

Reply:
Yes :-)
links are not displayed when we do not need them 👍